src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -1468,11 +1470,58 @@ internal void EnsureAnonymousTypeTemplates(CancellationToken cancellationToken)
         /// <param name="cancellationToken">To cancel the emit process.</param>
         public EmitResult Emit(
             Stream peStream,
+            Stream pdbStream,
+            Stream xmlDocumentationStream,
+            Stream win32Resources,
+            IEnumerable<ResourceDescription> manifestResources,
+            EmitOptions options,
+            CancellationToken cancellationToken)
+        {
```

Comment:
Yes, it could definitely be marked with `[EditorBrowsable(EditorBrowsableState.Never)]`